src/types/requests/sender.ts

```diff
@@ -0,0 +1,28 @@
+import { Sender } from '../models';
+import { RecursivePartial } from '../utils';
+
+/** @public */
+export interface GetAllSendersParameters {
+  /** Any text which is going to be used in conjunction with the searchtype if provided */
```

Comment:
```suggestion
  /** Any text which is going to be used in conjunction with the search type if provided */
```
nit - missing a space?

Reply:
This is copied directly from [OneSpan's documentation](https://community.onespan.com/products/onespan-sign/sandbox#/Senders/api.account.senders.get). 

I think it's best to keep it consistent. What do you think?

Reply:
Whats funny is if its an actual typo on their end and they correct it, their mistakes will live on through us lol.
I think its ok to stick with the source, even if the source might be 💩  All the more reason to be careful on publicly available code. You're mistakes can live on beyond the corrections. If it was code it should be one for one but documentation like this... if we want to this small correction is ok. Then again if it references a variable "searchtype", we'd be in trouble.  At least thats my take on it.

Reply:
Okay, now that I'm looking again, this description is the [exact same](https://community.onespan.com/products/onespan-sign/sandbox#/Packages/api.packages.get) from the `GET /api/packages` endpoint, which accepts a `searchtype` query parameter 🤨 

So this description here doesn't make much sense, unless it also accepts a `searchtype` query parameter

Reply:
Did another test and corrected the schema in [d01d12e](https://github.com/getbreathelife/onespan-sign-node/pull/26/commits/d01d12e27ccbadca3a6dfb7089432e8ed1e7793a)